zipline-gradle-plugin/src/main/kotlin/app/cash/zipline/gradle/ZiplinePlugin.kt

```diff
@@ -39,14 +45,53 @@ class ZiplinePlugin : KotlinCompilerPluginSupportPlugin {
     version = BuildConfig.KOTLIN_PLUGIN_VERSION,
   )
 
+  override fun apply(project: Project) {
+    super.apply(project)
+
+    val extension = project.extensions.findByType(KotlinMultiplatformExtension::class.java)
+      ?: return
```

Comment:
Eventually we'll need support for the plain JS plugin, too.

Reply:
Agreed.